modules/Kconfig.mcux

```diff
@@ -30,6 +30,11 @@ config HAS_MCUX_CCM
 	help
 	  Set if the clock control module (CCM) module is present in the SoC.
 
+config HAS_MCUX_LPC_CLOCKCONTROL
```

Comment:
This would be better named `HAS_MCUX_SYSCON`

Reply:
Shouldn't the devicetree node be named syscon as well?

Reply:
Yes

Reply:
I have updated this PR to use SYSCON